### DISCUSSION TITLE
Increase mouse responsiveness sensitivity when running on weston(wayland-server)

### DISCUSSION THREAD

**Discussion by warlice** [openfde/openfde#133] [Ideas]
Increase mouse responsiveness sensitivity by showing the wayland server mouse pointer（weston）and disabling the android mouse pointer

**Comment by ironlvxyy** [comment]
I think it should be possible to filter out the Android cursor layer from all the layers to be synthesized in hwcomposer for Android. Remove it from the synthesis, extract the image data of the Android cursor layer, and then transmit the Android cursor image data to Weston's cursor surface for display through the Wayland protocol.

**Reply by warlice** [reply]
ok，Allowing Weston, which serves as an X11 client, to directly display the mouse can reduce the latency of mouse displaying and is beneficial for improving mouse sensitivity.
